clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/download/DownloadStrategyResolverTest.spec.ts

```diff
@@ -15,22 +15,27 @@
 import { SettingsService } from "../../../../services/Settings";
 import { DownloadStrategyResolver } from "../../../../services/copybook/downloader/DownloadStrategyResolver";
 import { zoweExplorerMock } from "./getZoweExplorerMock.utility";
+import { e4eMock } from "./getE4EMock.utility";
+import { E4ECopybookService } from "../../../../services/copybook/E4ECopybookService";
 
 jest.mock("../../../../services/reporter/TelemetryService");
 
 describe("tests download resolver", () => {
   const downloader = new DownloadStrategyResolver(
     "storage-path",
     zoweExplorerMock,
+    e4eMock,
   );
   beforeEach(() => {
     jest.clearAllMocks();
   });
+  E4ECopybookService.getE4EAPI = jest.fn().mockResolvedValue(undefined);
```

Comment:
This looks like an unrealistic condition, on one hand the `downloader` has an e4eMock, on the other hand, there is no API that could have provided it.